x-pack/plugins/secops/public/components/timeline/timeline.tsx

```diff
@@ -111,19 +108,22 @@ export const Timeline = pure<Props>(
 
               <div data-test-subj="timeline">
                 {combinedQueries != null ? (
-                  <EventsQuery
+                  <TimelineQuery
                     sourceId="default"
-                    startDate={combinedQueries.queryProps.startDate}
-                    endDate={combinedQueries.queryProps.endDate}
-                    filterQuery={combinedQueries.queryProps.filterQuery}
+                    limit={itemsPerPage}
+                    filterQuery={combinedQueries.filterQuery}
+                    sortField={{
+                      sortFieldId: sort.columnId,
+                      direction: sort.sortDirection as Direction,
+                    }}
                   >
-                    {(resData: {}) => (
+                    {({ events, loading, totalCount, pageInfo, loadMore, updatedAt }) => (
                       <>
                         <Body
                           id={id}
                           columnHeaders={columnHeaders}
                           columnRenderers={columnRenderers}
-                          data={getOr([], combinedQueries.resParm, resData) as ECS[]}
+                          data={events as Ecs[]}
```

Comment:
You no longer need the `as Ecs[]` here and that is a _great_ thing as casts with `as xzy` can cause bad types to float around and other badness. It's best when a `as` cast isn't needed we don't add one or have one.

I would remove it on this line and the import at the top now that the import is not needed. TypeScript knows this type is Ecs :-)